### DISCUSSION TITLE
tag  subscribe

### DISCUSSION THREAD

**Discussion by tn987** [frangoteam/FUXA#321] [Ideas]
There are many points collected in the background. A view needs some tags. Can the  subscription mode be adopted to only transmit the required tags and reduce the amount of data sent. 

thanks!

**Comment by unocelli** [comment]
Hi,
Currently fuxa don't have a subscription functionality for tags communication backend/frontend. I agree with you that we need the feature.
There was a suggestion from someone to use [redis](https://redis.io/)

**Comment by tn987** [comment]
Hi，
When is it expected to be realized？

**Reply by unocelli** [reply]
Hi,
do you think it is urgent? on average, how many tags do you have in your projects?

**Comment by tn987** [comment]
hi,
In power plant, tags: 5000 ~ 20000

**Reply by unocelli** [reply]
Not bad...I know that it is a high priority task and we will look it in the near future.

**Comment by tn987** [comment]
proposal

Time series database is used to store data, such as “Apache iotdb” or “influxdb”

**Reply by unocelli** [reply]
about this there is already a issue #283
